BUILD.gn

```diff
@@ -84,7 +88,7 @@ grit("brave_resources") {
   ]
 
   deps = [
-    "app:brave_content_manifest_overlays",
+    "//chrome/app:chrome_content_manifest_overlays",
```

Comment:
I'm a little confused by this change. Why are switching to chrome_content_manifest_overlays and then patching them to add tor?

Reply:
original `app:brave_content_manifest_overlays` doesn't work at all.
It supposed to package `//chrome/utility:profile_import_manifest` but it never does.